.gitignore

```diff
@@ -16,3 +16,8 @@ __pycache__/
 
 ## Ignore pyenv configuration
 .python-version
+
+## Ignore ephemeral doc-tool output
+
+output.csv
```

Comment:
```suggestion
pageinfo.csv
```

Maybe? Just trying to match the default from the script, unless this is for something else.

Reply:
it **is** for something else, but i haven't finished making it yet. I'll leave it in for now :)

Reply:
Oooooh, mysterious! Can't wait to see :smile: